src/Private/template.txt

```diff
@@ -0,0 +1,18 @@
+function --name-- {
```

Comment:
These template files should go in a dedicated template folder as the build won't output them otherwise. I'll update the build script to pick the folder up and include it as it's output once I've merged this in.

Reply:
agreed as well, was gonna get to this today